guides/source/best_practice_create_form.md

```diff
@@ -20,7 +20,7 @@ mailer:
   mail_success_page: thanks
   mail_from: your.mail@your-domain.com
   mail_to: your.mail@your-domain.com
-  fields: [salutation, firstname, lastname, address, zip, city, phone, email, message]
+  fields: [salutation, firstname, lastname, address, zip, city, phone, email, message, contact_form_id]
```

Comment:
The `contact_form_id` is already an `attr_accessor` on the `Alchemy::Message` model

https://github.com/AlchemyCMS/alchemy_cms/blob/b6d8ceacd2ac1faa4a58b303194eaad079f1e9dc/app/models/alchemy/message.rb#L23